Empire/EmpireFirmware/EmpireFirmware.ino

```diff
@@ -300,105 +300,87 @@ void integralZone(double setpoint, double in,  int zone){
   
 }
 
+void clear_data() {
+  for(int i = 0; i < 10; i++) {
+    data_array[i] = 0;
+  }
+}
 
-void response_packet(bool success, double data) {
-  if(success) {
-    resp_buff[1] = 1;
+void response_packet(bool success, int controller_id, int data_len = 0) {
+  resp_buff[0] = 255; // header
+  resp_buff[1] = controller_id; // controller id
+  if(success) { // command byte used to signal success or fail
+    resp_buff[2] = 1; 
```

Comment:
Lets not use 1 or 0 because those commands are already being used. I mean technically we can just change the other commands its not that big of a deal its up to you. Also add these as #define SUCCESS_CODE and #define ERROR_CODE just so if we need to make changes to this variable we can.